09-group_average_sensors.py

```diff
@@ -0,0 +1,208 @@
+"""
```

Comment:
you should have use git mv here to keep track of last changes

Reply:
I tried to fix this by reverting the commit, using git mv, and then manually merging the changes again. Still GH doesn't seem to know about the history :(